intranet/apps/printing/views.py

```diff
@@ -349,17 +380,44 @@ def print_job(obj: PrintJob, do_print: bool = True):
         if obj.page_range:
             if not range_count:
                 raise InvalidInputPrintingError("You specified an invalid page range.")
-            elif range_count > settings.PRINTING_PAGES_LIMIT:
+            elif range_count > settings.PRINTING_PAGES_LIMIT_STUDENTS and obj.user.is_student:
```

Comment:
There are user types other than students and teachers. Apply the teacher limit to teachers and counselors (and, while we're at it, if the user is an admin they can get the teacher limit too). Apply the student limit to everyone else. Same logic for ratelimiting. No ratelimit for teachers and counselors. Ratelimit for everyone else (except admins).

Reply:
Actually, Justin reminded me `is_teacher` checks for teacher or counselor. So that's ok. But you will still need to cover other user types.

Reply:
I just made the first if statement check for printing admin perms or if they are a teacher, and anyone else gets the student limit because of the order the elifs appear in